lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -492,6 +492,17 @@ function _dosomething_reportback_get_permalink_copy_vars($user) {
  * Alters the signup form that is attached to permalink pages.
  */
 function dosomething_reportback_form_alter(&$form, &$form_state, $form_id) {
+
+  // Make sure a default run is added to the filter
+  if ($form['#id'] == 'views-exposed-form-reportback-files-page-1') {
+    if (empty($_GET['run_nid'])) {
+      $run = dosomething_helpers_get_current_campaign_run_for_user(arg(1));
+      if (isset($run)) {
```

Comment:
No I only want this to run when the filter is empty. If I did !empty( it would run even when a user specifies a filter